pkg/kubelet/status/manager.go

```diff
@@ -434,13 +435,19 @@ func (m *manager) needsReconcile(uid types.UID, status api.PodStatus) bool {
 		pod = mirrorPod
 	}
 
-	if isStatusEqual(&pod.Status, &status) {
+	podStatus, err := copyStatus(&pod.Status)
+	if err != nil {
+		return false
+	}
+	normalizeStatus(&podStatus)
```

Comment:
I think it's safer to normalize it here. Couldn't the api.Pod.Status be from the api server rather than the status manager?